src/mkfs.fat.c

```diff
@@ -74,6 +74,10 @@
 #define TRUE 1			/* Boolean constants */
 #define FALSE 0
 
+#ifndef OFF_MAX
+#   define OFF_MAX ~((off_t)1 << (sizeof(off_t) * 8 - 1))
+#endif
+
```

Comment:
Hi! Macro `OFF_MAX` is not defined in `src/common.h` which should be included in all files. It was part of changes in pull request https://github.com/dosfstools/dosfstools/pull/114

Can you rebase on top of master branch and remove above re-definition of `OFF_MAX`? After that I can merge this PR.

Reply:
Fixed.